manim/mobject/coordinate_systems.py

```diff
@@ -1022,6 +1022,7 @@ def create_axis(
         """
         new_config = merge_dicts_recursively(self.axis_config, axis_config)
         new_config["length"] = length
+        new_config["include_tick_at_origin"] = False
```

Comment:
This would probably be better suited for `axis_config`
```py
self.axis_config = {"include_tip": tips, "numbers_to_exclude": [0]}
```

Reply:
Done